storage/src/vespa/storage/storageserver/storagenode.cpp

```diff
@@ -247,11 +252,10 @@ StorageNode::updateUpgradeFlag(const vespa::config::content::UpgradingConfig& co
 void
 StorageNode::handleLiveConfigUpdate()
 {
```

Comment:
I think it wise signal the lock must be held in the signature.

Reply:
Agreed. The less we allow gun vs. foot interaction, the better.